src/main/java/core/basesyntax/Excavator.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax;
+
+public class Excavator extends Machine {
+
+    public Excavator(String machine) {
+        super(machine);
+    }

```

Comment:
The constructor for `Excavator` is currently accepting a `String` parameter named `machine`, which is then passed to the superclass `Machine`. However, the task requirements suggest that the message should indicate the type of machine working. Consider revising the constructor to not require any parameters and set a descriptive name within the class itself.